src/components/macro/macro-action-editor/tab/macro-key/macro-key.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
+
+import { KeyAction } from '../../../../../config-serializer/config-items/key-action';
+import { EditableMacroAction, MacroSubAction } from '../../../../../config-serializer/config-items/macro-action';
+import { KeypressTabComponent } from '../../../../popover/tab/keypress';
```

Comment:
`Tab` and `KeypressTabComponent` should be imported from `../../../../popover/tab/`